dsc/src/util.rs

```diff
@@ -406,16 +407,30 @@ pub fn get_input(input: &Option<String>, stdin: &Option<String>, path: &Option<S
     parse_input_to_json(&value)
 }
 
-pub fn set_dscconfigroot(config_path: &str)
+pub fn absolutize_and_set_dscconfigroot(config_path: &str) -> String
 {
     let path = Path::new(config_path);
-    let config_root = match path.parent()
-    {
-        Some(dir_path) => { dir_path.to_str().unwrap_or_default().to_string()},
-        _ => String::new()
+
+    // make path absolute
+    let full_path = path.absolutize().unwrap_or_default();
```

Comment:
what does `default` result in here?  should we error instead if we can't get an absolute path?

Reply:
Yes, processing error is better; updated.